module-presentation/src/main/kotlin/site/yourevents/invitation/api/InvitationApi.kt

```diff
@@ -51,4 +51,11 @@ interface InvitationApi {
     fun getGuestsByInvitation(
         @PathVariable invitationId: UUID
     ): ApiResponse<InvitationGuestResponse>
+
+    @Operation(summary = "초대장 발송자가 사용자인지 확인")
+    @GetMapping("/{invitationId}/is-sender")
```

Comment:
```suggestion
    @GetMapping("/{invitationId}/verify-sender")
```

다음과 같이 초대장 발송자를 `검증` 한다는 맥락을 주는 url은 어떨까요?

Reply:
처음에는 확인이라고 생각해서 `is`를 붙였는데, 정완님의 의견을 들어보니 해당 API의 의도가 검증의 로직에 더 가까운 것 같네요! 수정하도록 하겠습니다!